File: Python/PGS/LEVEL0/2의 영역/gunwoo.py
def solution(arr):
    answer = []
    cnt = arr.count(2)
    if cnt == 1:
        
        answer = [2]
        
    elif cnt == 0:
        answer = [-1]
        
    else:
        b = arr[::-1]
        answer = arr[arr.index(2): len(b)-b.index(2)]
            
    
    return answer

File: Algorithm/PGS/LEVEL1/나누어 떨어지는 숫자 배열/gunwoo.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        ArrayList<Integer> temp = new ArrayList<>();

        for (int value : arr) {
            if (value % divisor == 0) {
                temp.add(value);
            }
        }
        if (temp.isEmpty()) {
            temp.add(-1);
        }
        int[] answer = temp.stream().mapToInt(Integer::intValue).toArray();
        Arrays.sort(answer);

        return answer;
    }
}
